core/profile.c

```diff
@@ -578,6 +578,13 @@ struct plot_data *populate_plot_entries(struct dive *dive, struct divecomputer *
 		entry->tts = sample->tts.seconds;
 		entry->in_deco = sample->in_deco;
 		entry->cns = sample->cns;
+
+		if (dc->divemode == PSCR) {
+entry->o2sensor[0].mbar = 0;
+//			entry->o2sensor[0].mbar = sample->o2sensor[0].mbar; // For PSCR with O2 monitoring we assume O2 data from sensor[0]
+//			if ((dc->no_o2sensors == 0) && (sample->o2sensor[0].mbar)) // We assume only a single O2 sensor
+//				dc->no_o2sensors = 1;			// If there are indeed monitored o2 values, set the dc's no of sensors to 1
+		}
```

Comment:
New code, but commented out ... not sure you want this, or is an unintentional error.